src/screens/HomeScreen.js

```diff
@@ -148,7 +148,7 @@ export const HomeScreen = ({ navigation }) => {
           publicJsonFile="homeCarousel"
           refreshTimeKey="publicJsonFile-homeCarousel"
         />
-        <Widgets navigation={navigation} widgets={widgets} />
+        <Widgets navigation={navigation} widgetConfigs={widgets} />
```

Comment:
lets name the `widgets` also `widgetConfigs` here in `HomeScreen` for clearer reference?

Reply:
done